kafka_producer.js

```diff
@@ -28,15 +28,15 @@ app.use(session({
 }));
 
 // ExpressOIDC attaches handlers for the /login and /authorization-code/callback routes
-const oidc = new ExpressOIDC({
-    issuer: config.okta.domain + "/oauth2/default",
-    client_id: config.okta.clientId,
-    client_secret: config.okta.clientSecret,
-    loginRedirectUri: config.okta.host + "/authorization-code/callback",
-    appBaseUrl: config.okta.host,
-    scope: 'openid profile'
-});
-app.use(oidc.router);
+// const oidc = new ExpressOIDC({
+//     issuer: config.okta.domain + "/oauth2/default",
+//     client_id: config.okta.clientId,
+//     client_secret: config.okta.clientSecret,
+//     loginRedirectUri: config.okta.host + "/authorization-code/callback",
+//     appBaseUrl: config.okta.host,
+//     scope: 'openid profile'
+// });
+// app.use(oidc.router);
```

Comment:
Revert this.

Reply:
You can put in a config parameter to enable disable oidc. Put this code in if it is enabled. That way, if the param is not provided, oidc will be turned off.

Reply:
Added a config to enable or disable OIDC